apps/src/p5lab/utils.js

```diff
@@ -38,3 +40,15 @@ export function createSpriteCollider(sprite) {
     right: x + (width * scale) / 2,
   };
 }
+
+export function randomColor(p5) {
+  return p5
+    .color('hsb(' + randomInt(0, 359) + ', 100%, 100%)')
+    .toString('#rrggbb');
+}
+
+export function randomColorFromPalette(palette) {
+  return EFFECTS_PALETTES[palette][
+    randomInt(0, EFFECTS_PALETTES[palette].length - 1)
+  ];
+}
```

Comment:
Copied from https://github.com/code-dot-org/code-dot-org/blob/mike/effects-for-spritelab/apps/src/p5lab/poetry/commands/utils.js#L8-L16

Poetry doesn't seem to directly import these utils anywhere so I left them in both places. Possibly worth noting is that `randomInt`, which is also used for effects, was already defined in both utils files. This suggests that there is precedent here.

Reply:
I have a slight preference for only having these defined one place, even if we define them in `p5lab/utils` and have `poetry/commands/utils` import them and re-export them. That would look something like this in `apps/src/p5lab/poetry/commands/utils.js`:

```
import {
  randomInt,
  randomColor, 
  randomColorFromPalette 
} from '../../utils';

import {PALETTES} from '../constants';

// Adjusts P5's randomColorFromPalette function for Poetry's palette
export function randomColorFromPalettePoetry(palette) {
  return randomColorFromPalette(PALETTES[palette]);
}

export { randomInt, randomColor };

// Other stuff goes here...
```